doc/classes/AStar.xml

```diff
@@ -1,11 +1,23 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="AStar" inherits="Reference" category="Core" version="3.2">
 	<brief_description>
-		AStar class representation that uses 3d-vectors as edges.
+		An implementation of A* to find shortest paths among connected points in space.
 	</brief_description>
 	<description>
-		A* (A star) is a computer algorithm that is widely used in pathfinding and graph traversal, the process of plotting an efficiently directed path between multiple points. It enjoys widespread use due to its performance and accuracy. Godot's A* implementation make use of vectors as points.
-		You must add points manually with [method add_point] and create segments manually with [method connect_points]. So you can test if there is a path between two points with the [method are_points_connected] function, get the list of existing ids in the found path with [method get_id_path], or the points list with [method get_point_path].
+		A* (A star) is a computer algorithm that is widely used in pathfinding and graph traversal, the process of plotting short paths among vertices (points), passing through a given set of edges (segments). It enjoys widespread use due to its performance and accuracy. Godot's A* implementation use 3D points as vertices and use the Euclidean distance by default.
```

Comment:
Godot's A* implementation use -> Godot's A* implementation use**s**

Reply:
Haha sure 😝

"Godot's A* implementation uses points in three-dimensional space and Euclidean distances by default."